src/i18n/locales/en/translation.json

```diff
@@ -247,6 +247,16 @@
     "text_stopping": "Stopping",
     "button_show_report": "Show earnings report",
     "button_show_orderbook": "Show orderbook",
+    "current": {
+      "title": "Active Offer #{{ id }}",
```

Comment:
Does this ID mean much? I always get "Active Offer #0". Sounds like I can have multiple offers which I can't (right?) Maybe "Active Offer" is already enough? 🤔

Reply:
Changed according to the comment below -> `<nym>:<oid>`.